dev/conductor/core/test/next_test.dart

```diff
@@ -25,6 +25,7 @@ void main() {
   const String remoteUrl = 'https://github.com/org/repo.git';
   const String revision1 = 'd3af60d18e01fcb36e0c0fa06c8502e4935ed095';
   const String revision2 = 'f99555c1e1392bf2a8135056b9446680c2af4ddf';
+  const String revision3 = 'ffffffffffffffffffffffffffffffffffffffff';
```

Comment:
As always, @christopherfujino plan works out perfectly :-) (leaving me space to create an accurate revision 3)

Reply:
I had a feeling we would need this